templates/about.html

```diff
@@ -1,17 +1,26 @@
+<!-- prettier-ignore -->
 {% set active_page = "Help" %}
+<!-- prettier-ignore -->
 {% extends "base.html" %}
 
+<!-- prettier-ignore -->
 {% block tabs %}
+<!-- prettier-ignore -->
 {{ components.tabs([("faq", "FAQ", "active")]) }}
+<!-- prettier-ignore -->
 {% endblock %}
-
+<!-- prettier-ignore -->
```

Comment:
sorry i'm not quite familiar these so I don't know if this is the best way -- I have to ask prettier to ignore these lines since they are not normal HTML